root.sh

```diff
@@ -42,7 +42,7 @@ COMPILER_CC=cc
 COMPILER_CXX=c++
 COMPILER_LD=c++
 case $PKGVERSION in
-  v6-18*) 
+  v6-20*) 
```

Comment:
How about doing `v6-18*|v6-20*)` so that we can backport this, if needed?

Reply:
I think we should completely abandon v6-18.

Reply:
ok